app/templates/content/index-crown-hosting.html

```diff
@@ -0,0 +1,100 @@
+{% from 'macros/_breadcrumb.html' import breadcrumb %}
+{% extends "_base_page.html" %}
+
+{% block breadcrumb %}
+  {{ breadcrumb(crumbs) }}
+{% endblock %}
+
+{% block main_content %}
+
+    <div class="grid-row">
+      <div class="column-two-thirds">
+        <header class="page-heading">
+          <h1>
+            Physical datacentre space for legacy systems
+          </h1>
+        </header>
+      </div>
+      <div class="column-two-thirds">
+        <p class="service-summary-lede">
+          The Crown Hosting Data Centres framework is available to all central
+          government departments, arm’s length bodies and the wider public sector.
+        </p>
+        <p class="service-summary-lede">
+          Contact <a href="mailto:crownhosting@cabinetoffice.gov.uk">crownhosting@cabinetoffice.gov.uk</a>
+          to discuss datacentre hosting for your legacy systems.
+        </p>
+        <h2 class='summary-item-heading'>
```

Comment:
This spacing is fixed in 59d7ee5

![image](https://cloud.githubusercontent.com/assets/355079/7608571/b73144e0-f962-11e4-83b1-d2a6221793f8.png)